rholang/src/main/scala/coop/rchain/rholang/interpreter/errors.scala

```diff
@@ -81,8 +81,14 @@ object errors {
   final case class TopLevelFreeVariablesNotAllowedError(freeVars: String)
       extends InterpreterError(s"Top level free variables are not allowed: $freeVars.")
 
+  final case class ChannelConnectivesNotAllowedError(connectives: String)
```

Comment:
I don't think you need this one.
The `TopLevelConnectivesNotAllowedError` should be enough.